apps/web/components/getting-started/components/CalendarItem.tsx

```diff
@@ -20,8 +21,9 @@ const CalendarItem = (props: ICalendarItem) => {
       <InstallAppButtonWithoutPlanCheck
         type={type}
         render={(buttonProps) => (
-          <button
```

Comment:
Render button can't be a native button due to reliance on Button-only parameters later down the line (isMutation in some apps)